ga4gh/datamodel/__init__.py

```diff
@@ -70,9 +70,10 @@ def sanitizeVariantFileFetch(cls, contig=None, start=None, stop=None):
 
     @classmethod
     def sanitizeAlignmentFileFetch(
-            cls, referenceName=None, start=None, end=None):
-        if referenceName is not None:
-            referenceName = cls.sanitizeString(referenceName, 'referenceName')
+            cls, reference_name=None, start=None, end=None):
+        if reference_name is not None:
+            reference_name = cls.sanitizeString(reference_name,
```

Comment:
I usually prefer hanging indent to vertical alignment as it's more robust to code changes. Here for example, we'll need to realign the arguments after we snake_case the rest of the code, but with a hanging indent we won't need to touch the arguments.